### ISSUE TITLE
feat: allow providing node key using a file

### ISSUE THREAD

**Issue by jakubgs** [waku-org/nwaku#1593]
### Problem
Currently the only way to pass nod key to the node is using `--nodekey` flag which only accepts a string.

This is not optimal as it means you have to for example embed the node key in service definition which might have wider permissions that it warrants considering the sensitive nature of the key.

### Suggested solution
A separate flag - or path support for the current flag - would resolve this.

**Comment by mfw78** [comment]
FYI, one can actually pass the node key via an environment variable, notably `WAKUNODE2_NODEKEY`.

**Comment by jakubgs** [comment]
Yes, I'm already doing exactly that: https://github.com/NixOS/nixpkgs/pull/193679#discussion_r1127139016

But the issue with that is that if you for example mistype the key name it will fail silently and use a random key.
If you have an explicit key flag file and the file doesn't exist, or is empty, or has bad format, it will fail explicitly.
